### ISSUE TITLE
Document `Label` observable `text`

### ISSUE THREAD

**Issue by nandoconde** [JuliaPlots/Makie.jl#918]
Hello!

I noticed that there is no explicit comment in the docs about the ability to change the text of a `Label` object once it has already been created. 

Inspecting the object I saw that it contains an `Observable` called `text`. It may be good to have it documented and maybe an example in the `Label` page of the `Layoutables & Widgets` section?

I haven't made a PR because it may require to run the example and generate an image, but if you tell me I'll do it.

**Comment by SimonDanisch** [comment]
In general, anything in Makie should be updatable like this. The exceptions from that rule, and that specific rule may need better documentation.

**Comment by nandoconde** [comment]
I noticed the pattern you are saying, indeed. Mostly watching from other Layoutables docs, so I am not saying that it is an obscure or recondite feature.

But it would be nice having an explicit example from this exact feature in the `Label` doc page, I'd think.


Also, the pattern you are mentioning would be good to deocument explicitly. It is weird that I have to inspect properties using:
```julia
julia> fieldnames(typeof(fig.content[4]))
(:parent, :layoutobservables, :attributes, :elements)
julia> getproperty(fig.content[4], :elements)
Dict{Symbol, Any} with 1 entry:
  :text => Text{Tuple{String}}
```
And then
```julia
fig.content[4].text
```
just works. I like it A LOT that it works, but it'd be good to have it documented nonetheless.

Greetings!

**Comment by ffreyer** [comment]
If your goal is just to change the string displayed by the Label you're overcomplicating things. All the attributes you commonly want to adjust should be reachable with `layoutable.attribute_name`. So in this case

```julia
fig = Figure()
l = Label(fig[1, 1], "Hi")
l.text[] = "Hello"
```

**Comment by nandoconde** [comment]
Yeah, I am aware it is possible to do it like that. However, I am not always within the same context/namespace and therefore I may not be able to reach the binding to the Label created.

The way I do it now is:
```
content(fig[1,1]).text[] = "Hello"
```

The rest of the code was just to show that inspecting properties is not as straightforward as looking it up in the docs in this case.

**Comment by ffreyer** [comment]
Closing this since I don't really see anything to do
